client/components/main/layouts.styl

```diff
@@ -355,3 +355,15 @@ a
 @keyframes flexGrowIn
   from
     flex-basis: 0
+
+@media screen and (max-width: 800px)
+  #content
+    margin: 1px 0px 49px 0px
+    height: calc(100% - 96px)
```

Comment:
I never dared using `calc` because of browser support but that seem reasonable indeed http://caniuse.com/#search=calc. Cool!